src/app/pages/ParaTimesPage/TransactionConfirmation/__tests__/index.test.tsx

```diff
@@ -67,6 +77,31 @@ describe('<TransactionConfirmation />', () => {
     expect(navigateToSummary).not.toHaveBeenCalled()
   })
 
+  it('should render additional confirmation checkbox when transferring tokens to validator', async () => {
+    jest.mocked(useParaTimes).mockReturnValue({
+      ...mockUseParaTimesResult,
+      transactionForm: {
+        ...mockUseParaTimesResult.transactionForm,
+        recipient: 'validatorAddress',
+      },
+    } as ParaTimesHook)
+    when(useSelector as any)
+      .calledWith(selectValidators)
+      .mockReturnValue([{ address: 'validatorAddress' }])
+    render(<TransactionConfirmation />)
+
+    expect(screen.getByText('I confirm I want to transfer tokens to a validator address')).toBeInTheDocument()
+  })
+
+  it('should render additional confirmation checkbox when transferring tokens to foreign account', async () => {
+    when(useSelector as any)
```

Comment:
Ah, we should add a correctly-typed useSelector https://react-redux.js.org/using-react-redux/usage-with-typescript#define-typed-hooks and get rid of `as any`